python_modules/dagster-graphql/dagster_graphql/schema/pipelines/pipeline.py

```diff
@@ -229,6 +230,7 @@ class GrapheneRun(graphene.ObjectType):
     )
     startTime = graphene.Float()
     endTime = graphene.Float()
+    updateTime = graphene.Float()
 
     class Meta:
         interfaces = (GraphenePipelineRun,)
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"endLine":240}} -->
<i class='graphite__hidden'>[Re: line 240]<br/><br/></i>is that callsite above the only reason we need this?

Reply:
Yeah, updateTime is just a hack to get the `RunTime` component working without fetching stats